Bottom: 0ff4a98f70bbcdd4755bb46c5cdbad657698a8cf
Top:    78db8b5fb5df1ca51f07525b1b8831514ed9e5ec
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-12 09:44:38 -0500

Refresh of buildbot-master

---
diff --git a/roles/buildbot-master/tasks/main.yml b/roles/buildbot-master/tasks/main.yml
index ff25c6fb..3415bab6 100644
--- a/roles/buildbot-master/tasks/main.yml
+++ b/roles/buildbot-master/tasks/main.yml
@@ -33,12 +33,6 @@
     env: '{{ buildbot_env }}'
     ports: '{{ buildbot_port_args }}'
 
-- name: Grab ssh public key
-  command: >
-    docker exec '{{ buildbot_container_name }}' cat /home/buildbot/.ssh/id_rsa.pub
-  changed_when: false
-  register: output
-
 - name: Add buildbot_master host
   add_host:
     name: buildbot_master
